docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+
+.. figure:: figs/MultiRotor.png
+   :width: 80%
+   :name: fig:MultiRotor
+           
+   Definition of multiple rotors. 
+
+
+
+
+**Turbine geometry definition**
+
+Two turbine input formats are supported:
+
+- basic (`BasicHAWTFormat=True`): Basic horizontal axis wind turbine (HAWT) format. In this format, the turbine geometry is entirely determined by the number of blades (`NumBlades`), the hub radius (`HubRad`), the hub height (`HubHt`), the overhang (`Overhang`), the shaft tilt (`ShftTilt`) and the precone (`Precone`). The definition of each parameter follows the OpenFAST convention. Additionally, the user needs to provide the origin of the turbine base at `t=0` (`BaseOriginInit`).
+  An example of basic input is given below:
+
+.. code::
+
+    ----- Turbine(1) Geometry -------------------------------------------------------
+            True    BasicHAWTFormat(1) - Flag to switch between basic or generic input format {True: next 7 lines are basic inputs, False: Base/Twr/Nac/Hub/Bld geometry and motion must follow}
+           0,0,0    BaseOriginInit(1) - Coordinates of turbine base in global coordinates (m)
+               3    NumBlades(1)    - Number of blades (-)
+              3.    HubRad(1)       - Hub radius (m)
+          140.82513 HubHt(1)        - Hub height (m)
+              -7    Overhang(1)     - Overhang (m)
+              -6    ShftTilt(1)     - Shaft tilt (deg)
+              -4    Precone(1)      - Blade precone (deg)
+
+
+- advanced (`BasicHAWTFormat=False`): The position and orientation of the tower base, nacelle, hub, and individual blades can be arbitrarily defined. This can be used for HAWT and any other turbine concepts. 
+  The definition of the different frames are given in :numref:`fig:MultiRotor`.
+  The position (`BaseOriginInit`) and orientation (`BaseOriginInit`) of the turbine base frame are defined with respect to the global frame. Orientations are given using the values of three successive rotations (x-y-z Euler angles). If the base undergoes a motion, the orientation of the base frame will consists of the time-varying rotations followed by these initial rotations.
+
+  A flag indicating whether the turbine has a tower is given on the next line (`HasTower`). This flag currently affects the VTK outputs and does not have an impact on AeroDyn yet. The user still has to provide tower inputs data in AeroDyn for each turbine (see :numref:`ad_inputs_multirot`).
+  The next line indicates which projection AeroDyn is to use in its calculation. It is recommended to use `HAWTProjection=True` for HAWT, which is the default projection used in OpenFAST (projects on the coned-pitched axis). For other rotor concepts, set `HAWTprojection=False`.
```

Comment:
Add "BEMT" before "calculation".
Change "OpenFAST" to "AeroDyn".

Reply:
Some of the outputs of AeroDyn rely on this projection convention (BEMT or FVW).